tap_typeform/schemas/submitted_landings.json

```diff
@@ -34,6 +34,12 @@
     "hidden": {
       "type": ["null", "string"]
     },
+    "tags": {
+      "type": ["null", "array"],
+      "items": {
+        "type": ["string"]
```

Comment:
I think you misunderstood my last comment, the `null` should remain in the schema. That will allow null values to persisted if there are any. As it is written here if a null was returned by the API the transformer would throw an error.